.gitlab/packaging/deb.yml

```diff
@@ -0,0 +1,56 @@
+---
+
+.setup_deb_signing_key: &setup_deb_signing_key
+  - DEB_GPG_KEY=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh $DEB_GPG_KEY_SSM_NAME)
+  - printf -- "${DEB_GPG_KEY}" | gpg --import --batch
+  - export DEB_SIGNING_PASSPHRASE=$($CI_PROJECT_DIR/tools/ci/aws_ssm_get_wrapper.sh $DEB_SIGNING_PASSPHRASE_SSM_NAME)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏ isn't this fragment already centralized somewhere? It should be if it isn't.

Reply:
Good point, I was also surprised it wasn't, although it's only used in `deb.yml` so far, but copy pasting is... let's say not great :sweat_smile:

Reply:
Done!

Reply:
Ah I missed the comment in between.

I moved it to a common anchors file that's globally included, let me know if that works for you!